### PR TITLE
Show all stations

### PR DESCRIPTION
**Pull request by billfeng**
- The admin page now can show all stations if checkbox is ticked.
- There is currently no station data in database. needs to populate.

### CODE REVIEW

File: services/db/index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /* eslint-disable global-require */
 const Knex = require('knex');
 
-module.exports = function (config) {
+module.exports = (config) => {
   const knex = Knex({
```
Comment (kumilange) [inline services/db/index.js:5]: @bill-feng Thanks for fixing this! :)

File: server/routes/api.js [hunk 2]
```diff
@@ -10,8 +10,13 @@ module.exports = (services) => {
...
   router.get('/stations', async (req, res) => {
     try {
-      const stations = await services.db.stations.list();
-      res.status(200).send(stations);
+      const stations = (await services.db.stations.list()).map(station => { return {
+        id: station.id,
+        lat: parseFloat(station.latitude),
+        lng: parseFloat(station.longitude),
+      }});
+      console.log(stations);
```
Comment (kumilange) [inline server/routes/api.js:18]: @bill-feng Do you want to keep this console log?

File: src/components/Admin.jsx [hunk 3]
```diff
@@ -10,17 +10,33 @@ import Divider from 'material-ui/Divider';
...
+const stationIcon = {
+  path: 'M19.77 7.23l.01-.01-3.72-3.72L15 4.56l2.11 2.11c-.94.36-1.61 1.26-1.61 2.33 0 1.38 1.12 2.5 2.5 2.5.36 0 .69-.08 1-.21v7.21c0 .55-.45 1-1 1s-1-.45-1-1V14c0-1.1-.9-2-2-2h-1V5c0-1.1-.9-2-2-2H6c-1.1 0-2 .9-2 2v16h10v-7.5h1.5v5c0 1.38 1.12 2.5 2.5 2.5s2.5-1.12 2.5-2.5V9c0-.69-.28-1.32-.73-1.77zM18 10c-.55 0-1-.45-1-1s.45-1 1-1 1 .45 1 1-.45 1-1 1zM8 18v-4.5H6L10 6v5h2l-4 7z',
+  anchor: { x: 10, y: 10 },
+  fillColor: '#FFD441',
+  fillOpacity: 1.0,
+};
+
+const Admin = (props) => {
+  if (!props.stationsLoaded) {
+    console.log(props.showStations)
```
Comment (kumilange) [inline src/components/Admin.jsx]: @bill-feng Do you want to keep this console.log?

**Comment by kumilange** [comment]
@bill-feng Left comments on minor things. Other than that, LGTM!  👍

File: server/routes/api.js (hunk 2 above)
Comment (billfeng) [reply server/routes/api.js:18]: Yes. It's okay for the server to have console log.

File: src/components/Admin.jsx (hunk 3 above)
Comment (billfeng) [reply src/components/Admin.jsx]: Thanks, I fixed it!

**Comment by billfeng** [comment]
@kumiko-haraguchi Thanks, I fixed them! could you check again?

**Review by kumilange — APPROVED** [review]

**Pull request merged**